### PR TITLE
DOCS: Updates docs to reflect Launcher changes

### PR DESCRIPTION
**Pull request by cadihowley**
- updates screenshots to show Launcher grid view/changed settings tabs
- updates all settings pages to reflect new Launcher button terminology
- moves Engine tab content to Game tab page
- removes Engine tab from index
- fixes some broken refs/other small errors

### CODE REVIEW

**Comment by cadihowley** [comment]
Let me know if there's anything missing - or if there's anything else I should add while I'm at it!

**Review by criezy — APPROVED** [review]
The changes look good to me.

The new images look a bit blurry to me, especially when viewed on a non-Retina screen, but that was the case for most of the old images as well. I am not sure the change of format (using jpeg instead of png) helps though.

**Comment by cadihowley** [comment]
> The new images look a bit blurry to me, especially when viewed on a non-Retina screen, but that was the case for most of the old images as well. I am not sure the change of format (using jpeg instead of png) helps though.

Hmm I have to agree with you. Had some issues getting the latest build installed on my Linux machine so the screenshots are coming off my Android tablet, hence the jpg.... but as mentioned in Discord I think I'll be redoing ALL the screenshots on a mac anyway! Cheers for the feedback

**Comment by Thunderforge** [comment]
It looks like the launcher screenshots all have `2.7.0git` as their version.

If these are going to be the official docs for 2.6.0, they probably should say `2.6.0` instead (compare with `2.2.0` in many of the ones being replaced). I think it looks more professional that way as well since we're not using a daily build.

All that said, I don't know that I feel strongly enough about it to hold up the PR, especially if the screenshots are going to be redone in the near future to better accommodate retina screens.

File: doc/docportal/settings/cloud.rst
Comment (Thunderforge) [inline doc/docportal/settings/cloud.rst:9]: Typo, should be `necessary`

File: doc/docportal/use_scummvm/taskbar_integration.rst
Comment (Thunderforge) [inline doc/docportal/use_scummvm/taskbar_integration.rst:18]: FYI, #4024 will likely be backported to 2.6.0. This would change the instructions.

**Review by Thunderforge — CHANGES_REQUESTED** [review]

**Comment by Thunderforge** [comment]
GitHub reviews don't seem to let me add review comments to images, so here are a couple of things:

 * `Launcher/launcher_popup.jpg` has markings in the upper right for input type, which I think is an Android-exclusive thing. There's also a `launcher_popup.png` and I'm not sure if there should be both.
 * `settings/audio_game.png` has the audio override checkbox enabled in the old, but disabled in the new. It's a bit hard to cross-reference on GitHub with what the documentation is saying, but I want to make sure it's showing the right thing.
 * `settings/cloud.jpg` shows far less info in the new version than the old. Not sure if that's intentional.

File: doc/docportal/settings/game.rst
Comment (Thunderforge) [inline doc/docportal/settings/game.rst:165]: 🤣

**Review by sev- — COMMENTED** [review]
Very nice change. Speaking of 2.7.0git screenshots, what you can do, @cadihowley is use nightly build. Only the initial launcher screenshot is missing.

What I would also suggest to add is a sample with groups in the launcher list (not the icons view but the list view).

Also, it would be awesome to have couple of screenshots for "Update icons" dialog. (Options -> Misc, there is a button, you click it and a dialog appears).

File: doc/docportal/settings/cloud.rst
Comment (cadihowley) [reply doc/docportal/settings/cloud.rst:9]: > If these are going to be the official docs for 2.6.0, they probably should say `2.6.0` instead (compare with `2.2.0` in many of the ones being replaced). I think it looks more professional that way as well since we're not using a daily build.

I wasn't really intending these to be the docs for 2.6.0 necessarily - if they're going to be then I'll need a hand to create a pull request against branch-2-6. Not sure how to do that without making a mess of things! 

With regards to the screenshots, up until now (disregarding the initial v2.2 docs), screenshots have not been updated with each release. In an ideal world, any PR making changes to the GUI would have accompanying new screenshots and documentation. This would result in docs and screenshots that are always up to date in terms of functionality, without the screenshot launcher versions necessarily matching - but as you say, this does look less professional. The alternative is that prior to each release I create an entirely new set of screenshots, which would probably look best, but is obviously the most labour intensive option!

File: doc/docportal/use_scummvm/taskbar_integration.rst
Comment (cadihowley) [reply doc/docportal/use_scummvm/taskbar_integration.rst:18]: Thanks, I'll have a look and update accordingly!

**Comment by cadihowley** [comment]
>     * `Launcher/launcher_popup.jpg` has markings in the upper right for input type, which I think is an Android-exclusive thing. There's also a `launcher_popup.png` and I'm not sure if there should be both.

No, there shouldn't be! I thought I removed all the .jpg files on the second commit, but I was in a bit of a time crunch before going to work so I clearly missed one or two! 
  
>     * `settings/audio_game.png` has the audio override checkbox enabled in the old, but disabled in the new. It's a bit hard to cross-reference on GitHub with what the documentation is saying, but I want to make sure it's showing the right thing.

Good pickup - thanks 

>     * `settings/cloud.jpg` shows far less info in the new version than the old. Not sure if that's intentional.

Not entirely - the .jpg file is the wrong one, but the .png is also not the one that should be in there. I was drowning in screenshots and renamed the wrong one I think 🤣

**Comment by cadihowley** [comment]
> What I would also suggest to add is a sample with groups in the launcher list (not the icons view but the list view).
>
> Also, it would be awesome to have couple of screenshots for "Update icons" dialog. (Options -> Misc, there is a button, you click it and a dialog appears).

@sev- Consider it done!

File: doc/docportal/settings/cloud.rst
Comment (SupSuper) [reply doc/docportal/settings/cloud.rst:9]: > I wasn't really intending these to be the docs for 2.6.0 necessarily - if they're going to be then I'll need a hand to create a pull request against branch-2-6. Not sure how to do that without making a mess of things!

We can just cherry-pick this to branch-2-6 after it's merged.

> With regards to the screenshots, up until now (disregarding the initial v2.2 docs), screenshots have not been updated with each release. In an ideal world, any PR making changes to the GUI would have accompanying new screenshots and documentation. This would result in docs and screenshots that are always up to date in terms of functionality, without the screenshot launcher versions necessarily matching - but as you say, this does look less professional. The alternative is that prior to each release I create an entirely new set of screenshots, which would probably look best, but is obviously the most labour intensive option!

Maybe we can just hide the version number from the screenshots?

Comment (cadihowley) [reply doc/docportal/settings/cloud.rst:9]: >Maybe we can just hide the version number from the screenshots?

Maybe just blurring the version number would work?
launcher_list_group

**Comment by cadihowley** [comment]
@Thunderforge @sev- 
If you're happy with these commits for the 2.6 docs, please merge/cherry pick as required for branch-2-6. Sorry that it's last minute but unfortunately I won't be able to do any more before the branch is frozen!

**Comment by criezy** [comment]
Thank you for the updated pictures. We discussed a bit on Discord yesterday and decided to treat the documentation as a special case, and it will not be subject to the freeze this evening. That leaves us a few more days and I will have time on Sunday to look again at this pull request and make adjustments if needed before merging and backporting to the release branch.

File: doc/docportal/use_scummvm/taskbar_integration.rst
Comment (criezy) [inline doc/docportal/use_scummvm/taskbar_integration.rst:16]: I will modify this when merging as currently the taskbar is unable to use the icons.dat file that we can download from the Global Options. So icons still need to be downloaded manually.

**Review by criezy — APPROVED** [review]
That looks good to me and I think all the requested changes have been done. I noticed one issue with the taskbar doc update, but I will merge the PR manually and do the change myself during the merge.

File: doc/docportal/use_scummvm/taskbar_integration.rst
Comment (cadihowley) [reply doc/docportal/use_scummvm/taskbar_integration.rst:16]: Ahh thank you - I should've double checked this one, I just assumed!

**Comment by criezy** [comment]
I have now merged the changes manually and done a few additional changes:
 - Blurred the version in one additional image
 - Squashed a few commits together
 - Updated the documentation for the taskbar

The changes have also been backported to branch-2-6.

**Pull request closed without merging**
